docs/_includes/calendar.html

```diff
@@ -0,0 +1,37 @@
+<div id="calendar">
+    {% include section-header.html name="calendar" version=page.versions.calendar %}
+
+    <p>The calendar component is used as a building block for other components, such as date picker.</p>
+
+    <h3>Standard Calendar</h3>
+
+    <p>This component will stretch to the width of its container.</p>
```

Comment:
I would also emphasize that this example is showing a non-interactive calendar, with a non-interactive date range.